v2/app/service_discovery.go

```diff
@@ -49,6 +49,8 @@ func discoverServices(pc *parsectx.Context, result *parser.Result) []*Service {
 			sd.possibleServiceRoot(r.Package(), false)
 		case resource.PubSubSubscription:
 			sd.possibleServiceRoot(r.Package(), false)
+		case resource.AuthHandler:
```

Comment:
Is an auth handler the cause of a service to be created? I didn't think an auth handler was part of a service, but more library code